src/keys.ts

```diff
@@ -0,0 +1,28 @@
+import { purry } from './purry';
+
+function _keys<T>(object: { [key: string]: T } | ArrayLike<T>) {
+  return Object.keys(object) as string[];
+}
+
+/**
+ * Returns a new array containing the keys of the array or object.
+ * @param object
```

Comment:
parameters here say it has to be an `object`, but it can also be an Array as far as I can see. This is how it looks in the docs so it's a bit confusing imo:

<img width="580" alt="Screenshot 2021-07-25 at 20 50 17" src="https://user-images.githubusercontent.com/1021430/126910063-52d0a2d0-ae5a-428d-8d72-a28cccca7534.png">

see the deploy preview here: https://deploy-preview-137--trusting-lumiere-9c7fad.netlify.app/docs#values

I would say it would be good to rename `object` to maybe `source` and then describe here that it can be an object or an array?